### ISSUE TITLE
Subql-node v1.6.0 and v1.5.2-4 docker broken

### ISSUE THREAD

**Issue by wpank** [subquery/subql#1220]
### Prerequisites

* [x] Are you running the latest version(s)?
* [x] Did you check the [debugging guide](https://doc.subquery.network/academy/tutorials_examples/debug-projects.html#how-to-debug-a-subquery-project)?
* [x] Did you check the [FAQs](https://doc.subquery.network/faqs/faqs.html) and [Discussions](https://github.com/subquery/subql/discussions)?
* [x] Are you reporting to the correct repository?
* [x] Did you search for an existing issue or pull request?

### Description

I'm working on some indexing for Polkadot staking extrinsics, and yesterday everything was working fine. I try it again today (without changing any code) and get:

```
❯ yarn start:docker
Pulling postgres       ... done
Pulling subquery-node  ... done
Pulling graphql-engine ... done
Building with native build. Learn about native build in Compose here: https://docs.docker.com/go/compose-native-build/
nominator-subql_postgres_1 is up-to-date
Recreating nominator-subql_subquery-node_1 ... done

ERROR: for graphql-engine  Container "31d7676fdf6f" is unhealthy.
ERROR: Encountered errors while bringing up the project.
```

when trying to run the docker image. When inspecting the health status, seems it tries to curl some endpoint that isn't available:

```
❯ docker inspect --format "{{json .State.Health }}" 31d7676fdf6f | jq
{
  "Status": "unhealthy",
  "FailingStreak": 13,
  "Log": [
    {
      "Start": "2022-07-28T14:08:09.246773885+02:00",
      "End": "2022-07-28T14:08:09.366173339+02:00",
      "ExitCode": 22,
      "Output": "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n                                 Dload  Upload   Total   Spent    Left  Speed\n\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r  0    64    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\ncurl: (22) The requested URL returned error: 503\n"
    },

```

I tried a few different things, including a different rpc node, removing the dictionary, etc. What was weird is that `subql-node -f . ` worked fine. I then tried downgrading the `subql-node` docker image to `v1.5.2-4`, which resulted in the same thing, and then `v1.5.1`, which then worked. So this was something introduced introduced in `v1.5.2-4` and `v1.6.0`, which queries something that isn't returning right.

**Comment by cy6erninja** [comment]
Faced the same issue with onfinality/subql-node:latest (1.6.0). Downgrading to 1.5.1 indeed helped. Glad the fix is on its way 👍

**Comment by jiqiang90** [comment]
Checked the this seems due to polkadot update in v8.11.1 (https://github.com/polkadot-js/api/releases/tag/v8.11.1).

Now it has type generation for runtime call definitions (internal & external), https://github.com/polkadot-js/api/blob/e4a19dcefda8944a569ff15e52b8f610ee25803a/packages/api/src/base/Decorate.ts#L497

With user provide runtimes (https://github.com/polkadot-js/api/blob/master/packages/api/src/types/index.ts#L68),  when we use api.at() method to get current block and decorate our api call this will throw warning each time.

Will send a request to Polkadot for assistant.

**Comment by AtelyPham** [comment]
I'm facing the same problem, right now, I change the [health test endpoint](https://github.com/subquery/subql-starter/blob/v0.0.1/docker-compose.yml#L36) in `docker-compose.ymll` to `http://subquery-node:3000/health`, and it work for me. But it's best if [linked PR](https://github.com/subquery/subql/pull/1224) is merged.

**Comment by cy6erninja** [comment]
May I ask, how long does it take for the update to reach dockerhub?

**Comment by jiqiang90** [comment]
Hi @cy6erninja I think within this week will have a patch release for this, and new docker image will build and publish together

**Comment by stwiname** [comment]
This should be fixed with 1.6.1

**Comment by CocDap** [comment]
I m still facing this issue 
Node version: v20.8.0
